src/NUnitFramework/framework/Internal/TestFilter.cs

```diff
@@ -125,6 +125,9 @@ protected virtual bool MatchDescendant(ITest test)
         /// </summary>
         public static TestFilter FromXml(string xmlText)
         {
+            if (string.IsNullOrEmpty(xmlText))
```

Comment:
I tend to prefer to use `IsNullOrWhitespace`, but it isn't available in .NET 2.0. I could write an extension in compatibility. What do people think? Worth it for a hotfix?